src/Spryker/Configurator/DemoDateTimeConfiguratorPage/Theme/ConfiguratorApplication/src/services/http.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { ProductData } from './types';
+
+@Injectable()
+export class HttpService {
+    productData: ProductData;
+    constructor(private http: HttpClient) {}
+
+    getData(): Observable<ProductData> {
+        return this.http.get<{data: ProductData}>('/', {
+            params: new HttpParams().set('getConfigurationByToken', this.token)
```

Comment:
Params can be constructed from plain objects, please use them:
```suggestion
            params: { getConfigurationByToken: this.token }
```